### PR TITLE
FileExplorer tests

### PR DESCRIPTION
**Pull request by vbabich**
- `FileListItemEditor` - 100% coverage
- `FileUtils` - 88% coverage

### CODE REVIEW

File: packages/file-explorer/src/FileListItemEditor.test.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      onCancel,
+      onSubmit,
+      validate,
+    });
+    const input: HTMLInputElement = screen.getByRole('textbox');
+    userEvent.type(input, '123{enter}');
+    // Wait to resolve the validation promise
+    await act(async () => {
+      await Promise.resolve();
+    });
```
Comment (mofojed) [inline packages/file-explorer/src/FileListItemEditor.test.tsx:77]: Could put this in a convenience function or in `TestUtils.flushPromises()`

**Review by mofojed — DISMISSED** [review]

File: packages/file-explorer/src/FileListItemEditor.test.tsx (hunk 1 above)
Comment (vbabich) [reply packages/file-explorer/src/FileListItemEditor.test.tsx]: Updated

**Review by mofojed — CHANGES_REQUESTED** [review]
Now it's getting the `act` errors logged:
```
    console.error
      Warning: An update to FileListItemEditor inside a test was not wrapped in act(...).
      
      When testing, code that causes React state updates should be wrapped into act(...):
      
      act(() => {
        /* fire events that update state */
      });
      /* assert on the output */
      
      This ensures that you're testing the behavior the user would see in the browser. Learn more at https://reactjs.org/link/wrap-tests-with-act
          at FileListItemEditor (/home/bender/dev/deephaven/iris-oss/web-client-ui/main/packages/file-explorer/src/FileListItemEditor.tsx:28:3)

```

Might want to just wrap the flush in `TestUtils.flushPromises` with `act`

**Comment by vbabich** [comment]
Looks like we have a few files that use `TestUtils.flushPromises` in a non-RTL context. Adding `act` to `flushPromises` would add RLT as a new dependency in `TestUtils`, would it be better to define a local function with `act` in the file that needs it?

**Review by mofojed — APPROVED** [review]

**Pull request merged**